torch/_torch_docs.py

```diff
@@ -595,10 +595,10 @@ def merge_dicts(*dicts):
            r"""
 atan2(input, other, out=None) -> Tensor
 
-Returns a new tensor with the arctangent of the elements of :attr:`input`
-and :attr:`other`.
+Returns a new tensor with the arctangent of the angle between
+:math:`(\text{{other}}_{{i}}, \text{{input}}_{{i}})` and :math:`(1, 0)`.
 
-The shapes of :attr:`input` and :attr:`other` must be
+The shapes of ``input`` and ``other`` must be
```

Comment:
Why do you remove the use of `:attr:` here?

Reply:
Because `:attr:` is supposed to be used to refer to members (aka "attributes"). If there is an attribute with the same name, the hyperlink will point to the wrong page. See #25885

Reply:
Ho interesting, it actually just renders them with no link in this case. Thanks for catching that.